google/cloud/bigquery/table.py

```diff
@@ -47,6 +47,10 @@
 from google.cloud.bigquery.external_config import ExternalConfig
 from google.cloud.bigquery.encryption_configuration import EncryptionConfiguration
 
+# Types needed only for Type Hints
+from google.cloud import bigquery_storage  # pytype: disable=import-error
```

Comment:
As much as I think it'd be quite useful to depend on `google-cloud-bigquery-storage` being present, I don't think we wanted to make `google-cloud-bigquery-storage` a required dependency just for this feature.

How do other packages support cross-package types for their "extras"?

Reply:
It appears pytype supports this as well, will use it.

https://github.com/google/pytype/blob/master/docs/faq.md#how-do-i-write-code-that-is-seen-by-pytype-but-ignored-at-runtime